Update sensor.mold_indicator.markdown

tomlut opened the pull request:
Added optional 'name' configuration variable. I have confirmed this is supported but undocumented.

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

tomlut committed:
Update sensor.mold_indicator.markdown

Added optional 'name' configuration variable. I have confirmed this is supported but undocumented.

tjorim commented on source/_components/sensor.mold_indicator.markdown:
```diff
@@ -21,6 +21,7 @@ The sensor data may be used e.g., to signal bad air quality (too high air humidi
 # Example configuration.yaml entry
 sensor:
   - platform: mold_indicator
+    name: Mould Sensor
```
Please remove this line from the example, as per [the documentation](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration):

> This sample must contain only the required variables to make it easy to copy and paste it for users into their configuration.yaml file.

tjorim reviewed (changes requested):
Thanks for your contribution! Please consider removing the `name:`-key from the example configuration.yaml entry as explained in the comment below.

While updating this documentation anyway, would you mind changing the `Configuration variables:`-section to use the new {% configuration %} tags as explained [here](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration)? Thanks for helping out.

tomlut committed:
Update sensor.mold_indicator.markdown

Removed optional 'name' configuration variable from the example.

Changed the Configuration variables:-section to use the new {% configuration %} tags.

tomlut commented:
Done. Though looking back I think I made a mistake. The calibration_factor variable type is a float not an int. Though I don't see this as an option in the Create a new page documentation. Should it be a string instead?

tjorim commented:
Not all options are shown in the documentation (like boolean), as far as I know it can actually be anything. Feel free to use float instead of int, if it's wrong another dev will chime in.

tomlut committed:
Update sensor.mold_indicator.markdown

Changed calibration_factor variable type from int to float.

tomlut commented:
Thanks, I changed it to a float.

fabaff reviewed (approved):
Thanks :bird:
